modules/claims_api/app/workers/claims_api/claim_uploader.rb

```diff
@@ -49,9 +54,13 @@ def claim_upload_document(claim_document)
     end
 
     def service(auth_headers)
-      EVSS::DocumentsService.new(
-        auth_headers
-      )
+      if Flipper.enabled? :lh_bd_claim_uploader
+        ClaimsApi::BD.new
+      else
+        EVSS::DocumentsService.new(
+          auth_headers
+        )
+      end
```

Comment:
seems weird to have the flipper in both spots but not sure I have a better suggestion besides inlining it above.

Reply:
agreed. inline seems better